GPU/TPCFastTransformation/TPCFastTransform.h

```diff
@@ -370,7 +397,35 @@ GPUdi() void TPCFastTransform::Transform(int slice, int row, float pad, float ti
 
   if (mApplyCorrection) {
     float dx, du, dv;
-    mCorrection.getCorrection(slice, row, u, v, dx, du, dv);
+
+#if !defined(GPUCA_GPUCODE) && !defined(GPUCA_STANDALONE)
+    if (!mUseSlowCorrection) {
+#endif
+      mCorrection.getCorrection(slice, row, u, v, dx, du, dv);
+#if !defined(GPUCA_GPUCODE) && !defined(GPUCA_STANDALONE)
+    } else {
```

Comment:
If possible, could we reduce the usage of ifdefs a bit. E.g., the mUseSlowCorrection can be a normal member, then you can also just use it on the GPU.